winml/lib/Api.Ort/OnnxruntimeModel.cpp

```
@@ -37,8 +37,10 @@ HRESULT CreateFeatureDescriptors(
                             engine_factory->UseOrtApi());
 
     OrtTypeInfo* type_info;
-    RETURN_HR_IF_NOT_OK_MSG(feature_helpers->GetTypeInfo(ort_model, i, &type_info),
-                            engine_factory->UseOrtApi());
+    auto status = feature_helpers->GetTypeInfo(ort_model, i, &type_info);
+    if (status != nullptr) {
+      continue;
```

Curious why this is safe to ignore the error?